test/native/integration-test-helpers/setup-picker.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Platform } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import { fireEvent } from '@testing-library/react-native';
+import { ActionSheetIOS } from 'react-native';
+
+/**
+ * Sets up the Picker component for testing.
+ *
+ * @typedef {Object} PickerMockFunctions
+ * @property {Function}                                          selectOption Selects one of the options of the picker.
+ *
+ * @param    {import('@testing-library/react-native').RenderAPI} screen       A Testing Library screen.
+ * @param    {string[]}                                          options      Array with the options of the picker.
+ *
+ * @return {PickerMockFunctions} Picker functions.
+ */
+export function setupPicker( screen, options ) {
+	let selectOption = ( option ) => {
+		fireEvent.press( screen.getByText( option ) );
```

Comment:
This is the default action to select a picker option (i.e. it will be used on Android).